prime-router/src/main/kotlin/report/ReportService.kt

```diff
@@ -47,7 +48,13 @@ class ReportService(
      * @return List of ReportFile objects of the root reports
      */
     fun getRootReports(childReportId: ReportId): List<ReportFile> {
-        return reportGraph.getRootReports(childReportId)
+        return reportGraph.getRootReports(childReportId).distinctBy { it.reportId }
+    }
+
+    fun getReportsForStep(childReportId: ReportId, childIndex: Int, task: TaskAction): ReportFile? {
```

Comment:
kdoc? Also confusing because query only returns one but method name is getReport**s**?

Reply:
Yep, little copy pasta; this actually fetches a specific report for an item and action.  Will update the name